### DISCUSSION TITLE
MIT Change License

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#122] [Polls]
Just a poll to vote on a new license for the software. I personally don't mind the license though I'm sure someone in the future will bring up the conversation. "Why are you using MIT? Isn't this open software?" This is for that conversation.

**Comment by farfromrefug** [comment]
@jacobgonzales20 sure. what s te issue with MIT ? it is the most permissive one I I  not wrong

**Comment by 0-5-0** [comment]
This is a good explanation: https://choosealicense.com/ & https://choosealicense.com/licenses/

"The [MIT License](https://choosealicense.com/licenses/mit/) is short and to the point. It lets people do almost anything they want with your project, like making and distributing closed source versions."

"The [GNU GPLv3](https://choosealicense.com/licenses/gpl-3.0/) also lets people do almost anything they want with your project, except distributing closed source versions."

I guess it's if you want to recommend that someone publish their source and any improvements they might have made so that others can benefit versus allowing them to use your code even in a closed source format without them sharing any improvements.

**Reply by farfromrefug** [reply]
Yes that is the main difference. And i really dont care if someone decides to make a closed source proprietary version of what i do.
What i care about is that  the code is open source and that it can help others creat great softwares
